src/transition-jira-issues.ps1

```diff
@@ -85,28 +86,29 @@ try {
           -BaseUri $baseUri `
           -Jql $JqlToQueryBy `
           -IncludeDetails `
-          -MaxResults $MAX_ISSUES_TO_TRANSITION
+          -MaxResults ($MAX_ISSUES_TO_TRANSITION + 1)
     }
     catch [JiraHttpRequesetException] {
         if ($FailIfJiraInaccessible) { throw }
         Write-Warning "Jira might be down. Skipping transitions..."
         Write-Warning $_.Exception.MessageWithResponse()
         Write-Debug $_.ScriptStackTrace
     }
-
-    If ($issues.Length -eq 0 -And !$FailIfJiraInaccessible) {
+    
+    If ($issues.Length -eq 0 -And !$FailIfJiraInaccessible -And $CreateWarningNotices) {
```

Comment:
Do we still want to write something out if `$issues.Length -eq 0 -And !$FailIfJiraInaccessible` but `$CreateWarningNotices` is false? I'm thinking it might be nice to still have something in the output, but maybe it's still clear what the problem is without the warning 🤷

Reply:
This really is a way to defer the notice to job calling this action.  I am calling this action more than once in a single workflow and didn't want to see multiple notices when I could aggregate them into one.